### PR TITLE
chore: skip transactions root validation

### PR DESCRIPTION
**Pull request by chokobole**

### CODE REVIEW

**Comment by chokobole** [comment]
This should be merged after https://github.com/kroma-network/kroma/pull/65

**Review by 0xHansLee — APPROVED** [review]

**Review by Pangssu — APPROVED** [review]
LGTM

File: components/node/eth/output.go [hunk 1]
```diff
@@ -39,20 +39,24 @@ func (o *OutputResponse) ToPublicInput(ChainId *big.Int) (bindings.TypesPublicIn
...
+		withdrawalsRoot = *o.NextBlock.WithdrawalsHash
+	}
 	txHashes := make([][32]byte, len(o.NextTransactions))
 	for i, tx := range o.NextTransactions {
 		txHashes[i] = tx.Hash()
 	}
 	return bindings.TypesPublicInput{
-		Coinbase:         o.NextBlock.Coinbase,
+		BlockHash:        o.NextBlockRef.Hash,
+		ParentHash:       o.BlockRef.ParentHash,
```
Comment (0xHansLee) [inline components/node/eth/output.go:52]: You need to update as mentioned before.

**Review by 0xHansLee — CHANGES_REQUESTED** [review]

**Review by 0xHansLee — APPROVED** [review]

**Comment by 0xHansLee** [comment]
@chokobole Please check the conflict in `spec/challenge.md` once again 😃

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by Pangssu — APPROVED** [review]
Lgtm

**Pull request merged**